### PR TITLE
1673729 update all testapplications to latest framework and nuget packages

### PR DESCRIPTION
**Pull request by JLJKMD**
Updating framework and nuget packages for API, Main page, OpenId.MVCCore, SAML.MVCCore

### CODE REVIEW

File: KMD.Identity.TestApplications.OpenID.MVCCore/KMD.Identity.TestApplications.OpenID.MVCCore.csproj [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.15.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="5.0.6" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="5.0.6" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="8.0.4" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="8.0.4" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```
Comment (mci-kmd) [inline KMD.Identity.TestApplications.OpenID.MVCCore/KMD.Identity.TestApplications.OpenID.MVCCore.csproj:12]: Why was this suddenly needed?

Comment (JLJKMD) [reply KMD.Identity.TestApplications.OpenID.MVCCore/KMD.Identity.TestApplications.OpenID.MVCCore.csproj:12]: After updating everything I got an error message about it missing. 
We are using JsonConvert.DeserializeObject<T> in some places. 
Why it was not there before I dont know.

Comment (mci-kmd) [reply KMD.Identity.TestApplications.OpenID.MVCCore/KMD.Identity.TestApplications.OpenID.MVCCore.csproj:12]: Ok, no matter

**Review by gpikmddk — APPROVED** [review]
test if second approval is needed

**Pull request merged**